log-insights/setup/opentelemetry.mdx

```diff
@@ -31,18 +78,122 @@ In values.yaml file, update the section of `service`:
 </CodeBlock>
 
 This will create a service with the name `pganalyze-collector-otel-service`.
-Assuming you use [Fluent Bit](https://fluentbit.io/) as a telemetry agent,
-you can specify the created service as an OpenTelemetry HTTP endpoint, using the
-[OpenTelemetry output plugin](https://docs.fluentbit.io/manual/pipeline/outputs/opentelemetry):
+
+To create multiple OpenTelemetry endpoints, you can add multiple port entries under the `ports` section:
+
+<CodeBlock language="yaml">
+{`service:
+  create: true
+  name: pganalyze-collector-otel-service
+  type: ClusterIP
+  port: 4318
+  targetPort: 4318
+  ports:
+    - name: pganalyze-collector-otel1
+      port: 4318
+      targetPort: 4318
+    - name: pganalyze-collector-otel2
+      port: 4319
+      targetPort: 4319`}
+</CodeBlock>
+
+When there is a `ports` section, the Helm chart will ignore the settings 
+for any indidivual `port` and `targetPort` settings.
+
+
+## Fluent Bit configuration
+If you are using [Fluent Bit](https://fluentbit.io/) as a telemetry agent, you need to specify
+the correct INPUT and OUTPUT plugin settings to send logs to the pganalyze collector's OpenTelemetry 
+HTTP endpoint. Assuming that the Fluent Bit pod is correctly configured as a DeamonSet, the 
+following INPUT and OUTPUT configuration examples should successfully start sending logs from
+each Postgres pod to the pganalyze collector for processing. You will need to adjust your input
+path accordingly.
+
+<CodeBlock language="text">
+{`[INPUT]
+    name  tail
+    path  /var/log/postgresql/cluster-example-*_default_postgres*.log
+    Tag kube.*
+    multiline.parser docker, cri
+    DB /var/log/flb_kube.db`}
+</CodeBlock>
 
 <CodeBlock language="text">
-{`  [OUTPUT]
-    name  opentelemetry
-    match kube.*postgres*
-    host  pganalyze-collector-otel-service
-    port  4318`}
+{`[OUTPUT]
+  name  opentelemetry
```

Comment:
It'd be nice to use the same spacing as above (or use the same 2 spacing above, either way)

Reply:
Looks like you're using 4 spaces below, so maybe good to use 4 spaces here. Also maybe unify for the use of capitalization or not? (e.g. "Name" v.s. "name") - for INPUT section too.